clients/core-java/src/main/java/org/commonjava/indy/client/core/IndyClientHttp.java

```diff
@@ -607,7 +650,21 @@ public boolean validResponseCode( final int statusCode, final int[] responseCode
     public void close()
     {
         logger.debug( "Shutting down indy client HTTP manager" );
-        connectionManager.reallyShutdown();
+        if ( location != null )
+        {
+            try
+            {
+                factory.close();
```

Comment:
I've actually made a few changes locally that I'm trying to work in. Basically, I want to use jhttpc exclusively under the covers, and get rid of the code that inits httpclient bits directly. I didn't catch that in the PR when I was reviewing.